src/strategies/uniswapx_strategy.rs

```diff
@@ -26,14 +27,16 @@ use tracing::{error, info};
 use uniswapx_rs::order::{
     decode_order, encode_order, ExclusiveDutchOrder, OrderResolution, ResolvedOrder,
 };
+use std::env;
 
 use super::types::{Action, Event};
 
 const BLOCK_TIME: u64 = 12;
 const DONE_EXPIRY: u64 = 300;
-const REACTOR_ADDRESS: &str = "0xe80bF394d190851E215D5F67B67f8F5A52783F1E";
+const REACTOR_ADDRESS: &str = "0x6000da47483062A0D734Ba3dc7576Ce6A0B645C4"; 
+pub const SWAPROUTER_02_ADDRESS: &str = "0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45";
 pub const WETH_ADDRESS: &str = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2";
-pub const EXECUTOR_ADDRESS: &str = "TODO: Fill in swaprouter02 executor address";
+pub const EXECUTOR_ADDRESS: &str = env!("EXECUTOR_ADDRESS", "$EXECUTOR_ADDRESS is not set");
```

Comment:
really we should add it to args https://github.com/marktoda/uniswapx-artemis/blob/296304bee02e0d9d257786860b80076d6c6bbc1e/src/main.rs#L45

and pass in to strategy new(). With clap argparser you can support env as well